avm/res/container-service/managed-cluster/agent-pool/main.json

```diff
@@ -0,0 +1,386 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "2.0",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.21.1.54444",
```

Comment:
Please upgrade your Bicep version. Latest is `0.24.24`

Reply:
Same for the other templates. Just run `Set-AVMModule -ModuleFolderPath 'avm/res/container-service' -Recurse`